### PR TITLE
ETL Day2 Assignment Submission

### PR DESCRIPTION
**Pull request by Saphall**

### CODE REVIEW

File: Week3/Day2/docs/Documentation.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+con = databaseConnect('etl_day2')
+cur = con.cursor()
+
+....
+
+databaseDisconnect(con,cur)
+```
+
+Preety easy !
```
Comment (sumansta) [inline Week3/Day2/docs/Documentation.md:120]: Is it? :wink: I saw so many typos in this PR. Maybe you can improve on that. @Saphall

Comment (Saphall) [reply Week3/Day2/docs/Documentation.md:120]: Ok dai, I will fix the typos.

File: Week3/Day2/README.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+## ETL Design Assignment 2.
+
+> ## The file structure of this repo:
+```
+data/             # Folder containing the datasets in different formats.
+  *.json
+  *.csv
+  *.xml 
+docs/           # Folder containing .md files for assignment.
```
Comment (sumansta) [inline Week3/Day2/README.md:9]: Why did two of you had the same typo in this line? @Saphall :thinking: 
https://github.com/sanjeevbanmala/ETL/blame/master/README.md#L10

Comment (Saphall) [reply Week3/Day2/README.md:9]: Well, @silwalanish dai told everyone to take my documentation as reference, So mine typo was reflected in others maybe.

File: Week3/Day2/src/pipeline/database_connection.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import psycopg2
+
+def databaseConnect(database_name):
+    try:
+        return psycopg2.connect(
+            user='postgres',
+            password='admin',
+            host='localhost',
+            port='5432',
+            database=database_name
```
Comment (ShishirBhandari) [inline Week3/Day2/src/pipeline/database_connection.py:10]: find a way to not include these variables in git.
You can use google.

Don't use the database connection like this in the final project/assignment.

**Review by ShishirBhandari — APPROVED** [review]

File: Week3/Day2/src/pipeline/database_connection.py (hunk 3 above)
Comment (Saphall) [reply Week3/Day2/src/pipeline/database_connection.py:10]: ok dai I will use placeholder and manage by environment variables in final.

**Pull request merged**